### ISSUE TITLE
gem name is same as another gem

### ISSUE THREAD

**Issue by HuBandiT** [k3rni/ffi-locale#2]
this is the other gem with same name: https://github.com/seanohalpin/ffi-locale

since you pushed the new version, bundler picks this one up for me.

**Comment by k3rni** [comment]
This project was never pushed to rubygems, exactly because of that reason (there being at least one similar gem with conflicting name). Thus you have to use a `github` line in your Gemfile, or vendor it explicitely.

Can't do much about it except renaming. Got any ideas?

**Comment by HuBandiT** [comment]
Apologies, then I probably was seeing ghosts; I thought bundler pulled this one originally without any github spec, before I specified my repo with a git: opton to worki on my pull request, then dropped that option after you pushed out the new version - that's when I noticed it pulls in the wrong gem. So probably I never actually tried the first step but went into the second right away, and then misremembered.

In this case however, I would suggest enhancing the "Install" section with a Gemfile line example (and perhaps also mention and link to the clashing gem to hammer the point in that an explicit github option is needed).
